.circleci/config.yml

```diff
@@ -0,0 +1,98 @@
+version: 2.1
+orbs:
+    aws-cli: circleci/aws-cli@0.1.20
+jobs:
+  test-erbium:
+    docker:
+      - image: circleci/node:erbium
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: |
+            npm install
+            npm install jest-junit
+      - run:
+          name: Run tests
+          command: npm test -- --ci --runInBand --reporters=default --reporters=jest-junit
+          environment:
+            JEST_JUNIT_OUTPUT_DIR: ./reports/junit
+      - store_artifacts:
+          path: ./reports/junit
+      - store_test_results:
+          path: ./reports/junit
+      - persist_to_workspace:
+            root: .
+            paths: .
+  test-fermium:
+    docker:
+      - image: circleci/node:fermium
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: |
+            npm install
+            npm install jest-junit
+      - run:
+          name: Run tests
+          command: npm test -- --ci --runInBand --reporters=default --reporters=jest-junit
+          environment:
+            JEST_JUNIT_OUTPUT_DIR: ./reports/junit
+      - store_artifacts:
+          path: ./reports/junit
+      - store_test_results:
+          path: ./reports/junit
+      - persist_to_workspace:
+            root: . 
+            paths: .
+  deploy:
+    docker:
+      - image: circleci/python:3-node
+    steps:
+      - attach_workspace:
+          at: .
+      - aws-cli/setup
+      - run:
+          name: Install awscli
+          command: sudo pip install awscli

```

Comment:
you shouldn't need these if you've already added the cli orb and added the `aws-cli/setup` (line 55) afaik

Reply:
good spot! removed